### ISSUE TITLE
Add Thumbnail Navigation

### ISSUE THREAD

**Issue by mathetos** [mathetos/foogallery-owl-carousel-template#3]
This is a good request. It didn't originally seem possible, but there is an Owl plugin called "Linked Galleries" which does the job nicely:

https://github.com/OwlCarousel2/OwlCarousel2/issues/1378

Demo:
http://codepen.io/odahcam/pen/OXgWqg

This would require quite a big update to the template, so slotting this for version 1.5.

**Comment by odahcam** [comment]
I think we shouldn't add this to the core, but it would be nice to have this documented, I could do this, but there's no @daviddeutsch signal. 😢 

Just a few people use this feature, so keeping this on the core would add some useless weight for more of people. Adding the plugin to documentation with a well documented section would be nicer; it's my recomendation.

----

And thanks for using my example 😃 For any update there, you can call me.

**Comment by mathetos** [comment]
Hi @odahcam ya, for OwlCarousel.js that makes sense. But my Repo is for a WordPress plugin implementation. I'd have the ability to enable/disable this for my users per gallery so it wouldn't really add too much overhead.

Can you clarify what you mean by "there's no @daviddeutsch signal"? Thanks!

**Comment by odahcam** [comment]
Helo! Sorry, I was tinking I was on owlCarousel repo and realized that I wasn't only after posting my comment. xD

I appreciate that ur using daviddeutsh/my plugin and I liked the idea of including this plugin here. I can help you with this if you need.

I also said that there's no @daviddeutsch signal because it looks like owlCarousel is dying. :/ No merges, no new commits, no docs updates, etc.

**Comment by mathetos** [comment]
Ah ya... I'm actually using OwlCarousel v2, which does have commit activity:
https://github.com/OwlCarousel2/OwlCarousel2/commits/develop

If you wanted to submit a PR that implements your Linked Galleries feature I'd love that.

**Comment by odahcam** [comment]
Already did it! 🤔 

![image](https://cloud.githubusercontent.com/assets/3942006/23190991/0c8ad01a-f87a-11e6-8035-ded7096f2478.png)

I made a pull request to add the plugins for creating galleries w/ thumbnail navigation, but there's no reaction 😕, I don't even know if they are out of context.
